src/components/ToolMenu/Draw/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+import {
+  faDrawPolygon,
+  faGripLines,
+  faCircle,
+  faFont,
+  faSquare,
+  faPenToSquare,
+  faUpload,
+  faTrash
+} from '@fortawesome/free-solid-svg-icons';
+
+import {
+  FontAwesomeIcon
+} from '@fortawesome/react-fontawesome';
+
+import GeoJSON from 'ol/format/GeoJSON';
+
+import {
+  useTranslation
+} from 'react-i18next';
+
+import DeleteButton from '@terrestris/react-geo/dist/Button/DeleteButton/DeleteButton';
+import DrawButton from '@terrestris/react-geo/dist/Button/DrawButton/DrawButton';
+import ModifyButton from '@terrestris/react-geo/dist/Button/ModifyButton/ModifyButton';
+import SimpleButton from '@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton';
+import ToggleGroup from '@terrestris/react-geo/dist/Button/ToggleGroup/ToggleGroup';
+import UploadButton from '@terrestris/react-geo/dist/Button/UploadButton/UploadButton';
+import {
+  useMap
+} from '@terrestris/react-geo/dist/Hook/useMap';
+import {
+  DigitizeUtil
+} from '@terrestris/react-geo/dist/Util/DigitizeUtil';
+
+import './index.less';
+
+interface DefaultDrawProps { }
+
+export interface DrawProps extends Partial<DefaultDrawProps> { }
+
+export const Draw: React.FC<DrawProps> = (): JSX.Element => {
+  const {
+    t
+  } = useTranslation();
+
+  const map = useMap();
+
+  const onGeoJSONUpload = (geoJSONFile: any ) => {
+    // console.log(geoJSONFile);
+    const type = geoJSONFile.type !== 'application/geojson' ? geoJSONFile.type : 'NOT SUPPORTED';
+    if (type !== 'NOT SUPPORTED') {
+
+      var fileReader=new FileReader();
+
+      fileReader.onload = () => {
+        const geoJSONFeatures = new GeoJSON().readFeatures(fileReader.result);
+        geoJSONFeatures.forEach(feat => {
+          // TODO: Add some rule to have this projection always 4326 or 3857
+          feat.getGeometry()?.transform('EPSG:4326', 'EPSG:3857');
```

Comment:
GeoJSON should be EPSG:4326 always and we nigjt want to get the current projection from the map/view here.